configserver/src/main/java/com/yahoo/vespa/config/server/session/SessionPreparer.java

```diff
@@ -113,9 +113,8 @@ public ConfigChangeActions prepare(SessionContext context, DeployLogger logger,
             log.log(LogLevel.DEBUG, () -> "time used " + params.getTimeoutBudget().timesUsed() +
                     " : " + params.getApplicationId());
             return preparation.result();
-        } catch (OutOfCapacityException e) {
-            throw e;
-        } catch (IllegalArgumentException e) {
+        }
```

Comment:
We want to return "Out of capacity" in these cases, not "Invalid application".

Reply:
As `OutOfCapacityException` isn't an `IllegalArgumentException`, this is what happens now as well.

Reply:
Ah, of course. Good.